ui/app/ducks/swaps/swaps.js

```diff
@@ -92,6 +103,36 @@ const slice = createSlice({
     setMetamaskFeeAmount: (state, action) => {
       state.metamaskFeeAmount = action.payload
     },
+    setSwapsCustomizationModalPrice: (state, action) => {
```

Comment:
This is the anti-pattern that we discussed earlier - modelling a Redux slice as a collection of getters and setters.

See the[Redux Style Guide](https://redux.js.org/style-guide/style-guide/), specifically the ["Model Actions as Events, Not Setters"](https://redux.js.org/style-guide/style-guide/#model-actions-as-events-not-setters) section:

>**we recommend trying to treat actions more as "describing events that occurred", rather than "setters".** Treating actions as "events" generally leads to more meaningful action names, fewer total actions being dispatched, and a more meaningful action log history. Writing "setters" often results in too many individual action types, too many dispatches, and an action log that is less meaningful.

Reply:
Improvements in this regard were made in 4c8f00846

Reply:
Thanks, this looks much better!